go/service/pgp.go

```diff
@@ -162,7 +162,7 @@ func (h *PGPHandler) PGPImport(_ context.Context, arg keybase1.PGPImportArg) err
 		LogUI:     h.getLogUI(arg.SessionID),
 		SessionID: arg.SessionID,
 	}
-	eng, err := engine.NewPGPKeyImportEngineFromBytes(arg.Key, arg.PushSecret, nil)
+	eng, err := engine.NewPGPKeyImportEngineFromBytes(arg.Key, arg.PushSecret, h.G())
```

Comment:
I thought contextified still worked in this case to grab the global G if the passed `g` is `nil`